views/mixins/_postItem.pug

```diff
@@ -0,0 +1,8 @@
+//- this is basically our 'grid-item' for posts
+mixin postItem(post = {})
+  .post
+    .post--title
```

Comment:
We should add a link to the post here just to make it easy for us to navigate for now (I know you just slung this together to make sure it works haha)

Reply:
done